packages/admin-ui/src/DropdownMenu/components/DropdownMenuContent.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { cn, makeDecoratable } from "~/utils";
+
+const DropdownMenuContentBase = React.forwardRef<
+    React.ElementRef<typeof DropdownMenuPrimitive.Content>,
+    React.ComponentPropsWithoutRef<typeof DropdownMenuPrimitive.Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+    <DropdownMenuPrimitive.Content
```

Comment:
Shall we set the `collisionPadding` prop (https://www.radix-ui.com/primitives/docs/components/dropdown-menu#content)? 

I've observed that when the trigger is near the edge of the viewport, the menu content tends to be right next to it. To improve this, I suggest setting a default value for the property above to prevent any issues.